### PR TITLE
fullcontact app added

### PR DESCRIPTION
**Pull request by sarfaraz187**
App Request #126

### CODE REVIEW

**Review by Saif-Shines — APPROVED** [review]

File: Demo-Apps/fullcontact_app/app/app.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+/* On submit click. Fetch the contact's information from Enrich API */
+function getPerson() {
+  let fullName = $("#full_name").val(), email = $("#email").val(), phone_number = $("#phone_number").val();
+  fetchContactDetails(fullName, email, phone_number).then(function (payload) {
+    obj.isDbPresent ? updateProfile(fullName, email, phone_number) : saveProfile(fullName, email, phone_number);
+    displayModal(payload);
+    obj.profileId ? removeProfile(obj.profileId) : "";
+  }).catch(function (error) {
+    console.log(error);
```
Comment (ravirajsubramanian) [inline Demo-Apps/fullcontact_app/app/app.js:22]: Suggestion: Please use console.info or console.error statements to differentiate production logs from development logs, console.log statements.

File: Demo-Apps/fullcontact_app/app/modal.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+          </div>`
+          $(".container").append(html);
+        });
+      },function (error) {
+        console.log(error);
+      });
+    });
+  });
+</script>
+</html>
```
Comment (ravirajsubramanian) [inline Demo-Apps/fullcontact_app/app/modal.html:52]: Please add an empty line at the end-of-file.

Comment (ravirajsubramanian) [reply Demo-Apps/fullcontact_app/app/modal.html:52]: Also, please follow the same for all the files.

**Review by ravirajsubramanian — CHANGES_REQUESTED** [review]
Please remove /app/freshchat_logo.png as it is not used in the app.

File: Demo-Apps/fullcontact_app/app/app.js (hunk 1 above)
Comment (sarfaraz187) [reply Demo-Apps/fullcontact_app/app/app.js]: Thanks for the suggestion @ravirajsubramanian . changes made as requested do review.

File: Demo-Apps/fullcontact_app/app/modal.html (hunk 2 above)
Comment (sarfaraz187) [reply Demo-Apps/fullcontact_app/app/modal.html]: Thanks for the suggestion @ravirajsubramanian . changes made as requested do review.

**Review by ravirajsubramanian — APPROVED** [review]

**Comment by sarfaraz187** [comment]
Thank you @Saif-Shines and @ravirajsubramanian for your support in building this App.

**Comment by ravirajsubramanian** [comment]
Thank you to you for contributing to our repository. I hope it would have helped you as well.

If you have any feedback from your development experience, please share it with us on [our developer forum](community.developers.freshworks.com).

**Pull request merged**